tests/acceptance/test_examples.py

```diff
@@ -33,6 +33,8 @@ def test___shipping_example___run___no_errors(example_path: Path, system):
         pytest.skip("Example times out if there is no trigger signal.")
     if re.search(r"\binput\(|\bKeyboardInterrupt\b", example_source):
         pytest.skip("Example waits for keyboard input.")
+    if re.search(r"\bmatplotlib\b", example_source):
```

Comment:
need to add a tdms blob here, since I don't want to add those deps

Reply:
noted, tdms logging example is skipped from pytest.